Autopilot/Path Manager/MainBatterySensor.c

```diff
@@ -6,40 +6,44 @@
  */
 #include "main.h"
 #include "../Common/Common.h"
-#include "voltageSensor.h"
+#include "MainBatterySensor.h"
 
 
-char percent = 0;
-int currentVoltage = 0;
+int percent = 0;
```

Comment:
Although i see your point. You cant validate that a percentage is within a certain range if it overflows. In any case I dont think you should do the percentage conversion